velox/docs/functions/spark/math.rst

```diff
@@ -240,10 +240,10 @@ Mathematical Functions
     If ``x`` contains non-hexadecimal character, the function returns NULL.
     When ``x`` contains even number of characters, each pair is converted to a single byte. The number of bytes in the result is half the number of bytes in the input.
     When ``x`` contains a single character, the result contains a single byte whose value matches the hexadecimal character.
-    When ``x`` contains an odd number characters greater than 2, the first character is ignored, the remaining pairs of characters are converted to bytes, then zero byte is added at the end of the output. ::
+    When ``x`` contains an odd number characters greater than 2, the first character is decoded to one byte and put at index 0 of the output, the remaining pairs of characters are converted to bytes, and put start from index 1 of the output. ::
```

Comment:
This behavior matches only Spark 3.5 or earlier? Should we mention that this behavior match Spark Xxx, but not earlier?

Reply:
Spark fix it since 3.3.2. 

> Should we mention that this behavior match Spark Xxx, but not earlier?

I'm ok for this, thanks!

Reply:
> Spark fix it since 3.3.2.

Got it. Thank you for clarifying.